crossfit/backend/cudf/series.py

```diff
@@ -90,7 +91,9 @@ def create_list_series_from_1d_or_2d_ar(ar, index):
     else:
         return RuntimeError(f"Unexpected input shape: {ar.shape}")
     data = as_column(ar.flatten())
-    offset_col = as_column(cp.arange(start=0, stop=len(data) + 1, step=n_cols), dtype="int32")
+    offset_col = as_column(
+        cp.arange(start=0, stop=len(data) + 1, step=n_cols), dtype=np.dtype("int32")
```

Comment:
Using `DtypeObj = Union["ExtensionDtype", np.dtype]` from [here](https://github.com/rapidsai/cudf/blob/fa5dd546585eae884ba30d9365f84a8091eac164/python/cudf/cudf/_typing.py#L26), I set `dtype` to use NumPy.

Reply:
Thanks, this is a good find.